### PR TITLE
clusterversion: bump MinSupported to 24.3

### PR DESCRIPTION
**Pull request by RaduBerinde**
Epic: REL-1292
Release note: None

### CODE REVIEW

**Comment by RaduBerinde** [comment]
Friendly ping. I'd love to merge this soon to avoid merge conflicts.

File: pkg/ccl/cliccl/testdata/ear-list [hunk 1]
```diff
@@ -8,13 +8,13 @@ list
 000004.log:
   env type: Data, AES128_CTR
   keyID: bbb65a9d114c2a18740f27b6933b74f61018bd5adf545c153b48ffe6473336ef
-  nonce: e8 33 ee 2f ba ff 71 dd 29 20 97 62
```
Comment (dt) [inline pkg/ccl/cliccl/testdata/ear-list:11]: why does this change?

**Review by dt — APPROVED** [review]

**Review by fqazi — APPROVED** [review]
:lgtm:

> Reviewed 75 of 75 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/136509#-:-ODM9A7H-QHOxtneQNsx:bnfp4nl)* status: <span>:shipit:</span> complete! 1 of 0 LGTMs obtained (waiting on @arulajmani, @celiala, @jbowens, @RaduBerinde, @rail, @rytaft, and @xinhaoz)

**Review by rail — APPROVED** [review]
LGTM, :shipit:

**Comment by RaduBerinde** [comment]
TFTRs!

bors r+

**Pull request merged**